packages/discord.js/src/client/actions/MessageReactionAdd.js

```diff
@@ -22,7 +22,7 @@ class MessageReactionAdd extends Action {
     if (!user) return false;
 
     // Verify channel
-    const channel = this.getChannel(data);
+    const channel = this.getChannel({ id: data.channel_id, guild_id: data.guild_id, user_id: data.user_id });
```

Comment:
Channels have no user_id
```suggestion
    const channel = this.getChannel({ id: data.channel_id, guild_id: data.guild_id });
```

Reply:
https://github.com/discordjs/discord.js/blob/432e9b8425311f132e3970d238005c1166215dbb/packages/discord.js/src/client/actions/Action.js#L36

Reply:
Oh true, my bad…